corehq/apps/es/management/commands/elastic_sync_multiplexed.py

```diff
@@ -81,6 +83,43 @@ def start_reindex(self, cname, reindex_batch_size=1000, requests_per_second=None
             + f"\"grep '{task_number}.*ReindexResponse' /opt/data/elasticsearch*/logs/*.log\""
             + "\n\n")
 
+    def reindex_partial_index(self, cname, domain):
```

Comment:
nit: Can we make `cname` variable name more descriptive, something like `source_cname` or `destination_cname`?